src/utils/object.ts

```diff
@@ -7,7 +7,7 @@ import {Md5} from 'ts-md5/dist/md5';
  * @param objects
  */
 export function createChecksum(object: object | [], objects: any[] = []): string {
-    if(!object){
+    if (!object) {
         return ""
     }
```

Comment:
Brauchen wir das?

Reply:
Das ist Code der an sich schon vorher drin war und jetzt nur durch drücken von Strg + Alt + L mit zusätzlichen Leerzeichen umformatiert wurde

Reply:
Ich meine den generellen Check auf not null? Sollte ja die Prüfsummenbildung auf `null` oder `undefined` nicht beeinflussen.

Reply:
Ja ok, ist wohl richtig, ich dachte es wird fürs recursive sort gebraucht, aber das scheint nicht der Fall zu sein, wie ich das sehe

Reply:
md5.js?ee0c:254 Uncaught (in promise) TypeError: Cannot read property 'length' of undefined

die hash-Funktion scheint wider erwarten nicht mit undefined umgehen zu können. Entsprechend wäre es sinnvoll die Prüfung unsererseits mit drin zu lassen